clickhouse_connect/driver/insert.py

```diff
@@ -198,3 +199,12 @@ def _convert_numpy(self, np_array):
                 data[ix] = data[ix].tolist()
         self.column_oriented = True
         return data
+
+    def start_column(self, name: str):
+        super().start_column(name)
+        self._column_name = name
+
+    def make_data_error(self, error_message: str) -> DataError:
+        if self._column_name is not None:
```

Comment:
I don't know if it's possible for us to be handling a column insert without start_column() having been called, but I handled `None` just in case

Reply:
It's not possible, so it's safe to remove the None check.